src/css/main.scss

```diff
@@ -106,13 +106,44 @@ $blue-epsilon: #bce7f7;
 
 .form-group {
   &.required {
-  .col-form-label:after {
-    content: "*";
-    color: red;
+    .col-form-label:after {
+      content: "*";
+      color: red;
+    }
   }
+}
+
+.create-form {
+  border-radius: 5px;
+  border: 1px solid $gray-alpha;
+  padding-top: 50px;
+  padding-bottom: 30px;
+  box-shadow: 3px 3px whitesmoke;
+
+  .form-group {
+    margin-left: 30%;
   }
+  .btn {
+    margin-top: 40px;
+  }
+}
+
+.btn {
+  margin-left: 10px;
+  margin-right: 10px;
 }
 
+.movement-number {
```

Comment:
I suggest adding a bit of margin on bottom here (~15px) - please check it out and decide if this looks good or not (imho it looks better, than without margin).